README.rst

```diff
@@ -184,19 +184,64 @@ Here are a snapshot of values in ``all_styles["bg-style"]`` sheet which you may
         'figure.facecolor': 'w',
         'savefig.bbox': 'tight'
 
+Get individual color and color name
+-----------------------------
+
+You may select a specific hex color code or retrieve the color name from `Colors`: ::
+
+        from bg_mpl_stylesheets.colors import Colors
+
+        # Get the name of the color
+        Colors.bg_blue.name  # "bg_blue"
+
+        # Get the hex color code  
+        Colors.bg_blue.value  # "#0B3C5D"
+
+        # Get the color name from a hex code
+        color_name = Colors.get_color_name("#0B3C5D")  # Example: 'bg_blue'
+
+        # Get a list of all bg-style colors
+        bg_colors = Colors.get_bg_colors()
+
+Use a specific color to plot
+----------------------------
+
+You may select the specific color to plot: ::
+
+        import matplotlib.pyplot as plt
+        from bg_mpl_stylesheets.colors import Colors
+
+        x = [0, 1, 2, 3, 4, 5]
+        y = [i ** 3 for i in x]  # Example data: y = x^3
+
+        # Get hex color code
+        bg_blue_hex = Colors.bg_blue.value
+
+        # Get color name
+        bg_blue_name = Colors.bg_blue.name
```

Comment:
here we are teaching people somewhat ugly coding practices so I would remove these `bg_blue_hex` etc. variables.

We do want to let people know how to access the name and hex values, but let's do that with words, not with bad code....

The beauty of the enum is how easy and `pythonic` it is to access the names and values.

I am not totally sure why we don't use a dict, but it is ok

Reply:
Yup value and name directly accessed in the example.